src/Raven.Client/Documents/Replication/ReplicationNode.cs

```diff
@@ -115,6 +115,17 @@ public bool IsEqualTo(ReplicationNode other)
                    DictionaryExtensions.ContentEquals(SpecifiedCollections, other.SpecifiedCollections);
         }
 
+        public int CompareTo(ReplicationNode other)
+        {
```

Comment:
Need a comment here explaining why we just need a stable sort, not caring about the actual values.